src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +19,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
Consider wrapping the use of the `SessionFactory` in a try-with-resources statement or ensure that you close it at the end of your application to prevent potential resource leaks.